src/onchain/swap.rs

```diff
@@ -232,7 +234,7 @@ impl Swap {
 
         let lsp_fees = self
             .support
-            .calculate_lsp_fee_for_amount(send_amount_sats)?
+            .calculate_lsp_fee_for_amount(send_amount_sats, None)?
```

Comment:
This logic needs to be fixed anyway (we should use lsp fee params provided as input -  let's fix in another PR), but I would use here the new swap version of the method.